docs/sources/api_docs/_index.md

```diff
@@ -258,6 +259,48 @@ $ curl -G -s http://localhost:3200/api/search/tag/service.name/values  | jq
 }
 ```
 
+### Search tag values V2
+
+This endpoint retrieves all discovered values and their data types for the given TraceQL identifier.  The endpoint is available in the query frontend service in
+a microservices deployment, or the Tempo endpoint in a monolithic mode deployment. This endpoint is similar to `/api/search/tag/<tag>/values` but operates on TraceQL identifiers and types.  See [TraceQL](../traceql/) documention for more information. The following request will return all discovered service names.
```

Comment:
Clear explanation of what this does and it has link! Best practice to avoid using future tense, when possible. 
```suggestion
a microservices deployment, or the Tempo endpoint in a monolithic mode deployment. This endpoint is similar to `/api/search/tag/<tag>/values` but operates on TraceQL identifiers and types. See [TraceQL](../traceql/) documention for more information. The following request returns all discovered service names.
```